web/app_dev.php

```diff
@@ -18,7 +18,10 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+/**
+ * @var Composer\Autoload\ClassLoader $loader
+ */
+$loader = require __DIR__.'/../app/autoload.php';
```

Comment:
Once upon a time, fabpot didn't want the front controllers to work differently. In practice, I like what you've done here. But we would need to be pretty much absolutely sure that there could never be (and has never been any reported) differences between these. Can we be that sure?